src/pages/version/ParametersModal/index.tsx

```diff
@@ -26,23 +35,34 @@ export const ParametersModal: React.FC<ParametersProps> = ({ parameters }) => {
         </MetadataItem>
       )}
       <DisplayModal
+        data-cy="parameters-modal"
         open={showModal}
         setOpen={setShowModal}
+        size="large"
         title="Patch Parameters"
-        data-cy="parameters-modal"
       >
-        {parameters?.map((param) => (
-          <StyledBadge key={`param_${param.key}`}>
-            {param.key}: {param.value}
-          </StyledBadge>
-        ))}
+        <OverflowContainer>
+          <BaseTable table={table} />
```

Comment:
Imo the striping looks good here
```suggestion
          <BaseTable table={table} shouldAlternateRowColor />
```
<img width="983" alt="image" src="https://github.com/evergreen-ci/spruce/assets/9298431/96be8fcf-e88f-4f78-a3ce-a31cdf389ae4">